docker/cuda.Dockerfile

```diff
@@ -69,6 +69,9 @@ python -m pip install --no-cache-dir .
 python -m pip install --no-cache-dir "${SDK_REQUIREMENT}"
 EOF
 
+# Prefer to use pip, python, and other binaries from the virtual env.
+ENV PATH="/opt/orquestra/venv/bin:$PATH"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Same here